lib/stripe/discount.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module Stripe
+  class Discount < StripeObject
+    OBJECT_NAME = "discount".freeze
+  end
+end
```

Comment:
Doesn't introducing this class make it a breaking change for `discount` objects that are suddenly `Discount` and not `StripeObject` or whatever they were before?

Reply:
They're still a `StripeObject`. The only way this would break is if for some reason your check looked like `r.class == StripeObject` (i.e. the object's class is _exactly_ `StripeObject`) instead of the normal thing to do which is `r.is_a?(StripeObject)` (i.e. the object's class is `StripeObject` or one of its descendants).